pages/documentation/pythnet-price-feeds/price-service.mdx

```diff
@@ -1,5 +1,7 @@
 # Price Service
 
+** Price Service is going to deprecate soon. Pleaes consider using [Hermes](hermes), a complete rewrite of Price Service with the same API. **
```

Comment:
I have a bunch of random thoughts on this which are sort of unorganized and contradictory, so take them for what they're worth.

1. I think this warning can convey the same message in a much more compelling way to users. There are many benefits of using Hermes that aren't listed here. I think you could write this in a way where the users come out *excited* to switch, instead of feeling like "oh man here's another chore to do". The way to do that would be to say something like "As part of the Perseus Network Upgrade (link to blog), the Price Service has been replaced by Hermes. Please switch to Hermes in order to obtain the benefits of the upgrade, including increased update frequency and reduced verification cost. The Hermes API is fully-backward compatible so migrating simply requires changing a URL."

2. Do we even have to put in a deprecation warning here? We can change xc-mainnet etc. to point to hermes under the hood. if we do that, then users don't really have to migrate at all.

3. couple minor grammatical notes here:
"The Price Service will be deprecated soon."

And I don't think they can "consider" switching to hermes if it's going to be deprecated. They have to switch.

I also don't like "complete rewrite" part because that's not a benefit for the user. it's an implementation detail. You're missing an opportunity to explain to users why they should be switching and what value they get by doing it.

Reply:
yeah i agree with all of your feedbacks. Thanks. I think we can eventually replace it with a proper notice. That's easier!